### ISSUE TITLE
Bug. Boxplot of different color is not possible

### ISSUE THREAD

**Issue by Dzheyhan-tgmg** [ecomfe/echarts#6758]
### One-line summary [问题简述]

Hello,

I'm trying to set the label color for a Graphic but boxplot graphs is not generating when I try to add a custom series where the data entries are obj. Аlthough the documentation says it is possible.  https://ecomfe.github.io/echarts-doc/public/en/option.html#series-boxplot.data
For example, I may want the label color to be for Boxplot-1 red for Boxplot -2 green and for Boxplot -3 blue

Thanks in advance.

### Version & Environment [版本及环境]
+ ECharts version [ECharts 版本]: 3.7.1
+ Browser version [浏览器类型和版本]: Chrome
+ OS Version [操作系统类型和版本]: Ubuntu 16.04

![boxplots](https://user-images.githubusercontent.com/26247184/30876650-88b49b80-a2f7-11e7-8563-40a5139db3c9.jpg)

### Expected behaviour [期望结果]

Boxplot-1 with a red color   Boxplot-2  with a green  color and  Boxplot-3 with a blue color

### ECharts option [ECharts配置项]

```javascript
option = {
	yAxis: {
    	        type: 'category',
    	        data: ['Boxplot-1',  'Boxplot-2',  'Boxplot-3']
	},
	xAxis: {
    	        type: 'value'
	},
	series: [
    	{
        	name: 'boxplot',
        	type: 'boxplot',
        	data: [
            	         [713.75, 807.5, 810, 870, 963.75],
            	         [713.75, 807.5, 810, 870, 963.75],
            	        {
                	        value: [713.75, 807.5, 810, 870, 963.75],
                	        itemStyle: {
                    		      color: ' #0d0ded ',
                    		      borderColor: '#000000'
                		}
            	         },
        	],
    	},
    	{
        	name: 'outlier',
        	type: 'scatter',
        	data: [ [650, 'Boxplot-1'], [500, 'Boxplot-2'], [620, 'Boxplot-3'] ]
    	}
	]
}

```

**Comment by 100pah** [comment]
It is a bug fixed on 3.7.2, which has been released yesterday.

**Comment by Dzheyhan-tgmg** [comment]
thanks
